scripts/cli/util/distributePOLYX.js

```diff
@@ -0,0 +1,59 @@
+// Set options as a parameter, environment variable, or rc file.
+require = require("esm")(module /*, options*/);
+module.exports = require("./init.js");
+
+let { reqImports } = require("./init.js");
+const minimist = require('minimist');
+const args = minimist(process.argv.slice(2), {
+    string: 'account'
+});
+
+async function main() {
+    const api = await reqImports.createApi();  
+    const account = await reqImports.generateEntityFromUri(api, args.account);  
+    const amount = args.amount; 
+
+    let empty_did = "0x0000000000000000000000000000000000000000000000000000000000000000"
+    const listOfDids = await api.query.identity.didRecords.entries();
+    let all_dids = new Array();
+    
+    for (let i = 0; i < listOfDids.length; i++) {
+      let pk = listOfDids[i][1]['primary_key'];
+      let did = await api.query.identity.keyToIdentityIds(pk);
+      if (did.toString() != empty_did) {
+        all_dids.push(did);
+      }
+    }    
+    
+    await batchAtomic(api, account, all_dids, amount);
+    process.exit();
+}
+
+async function batchAtomic(api, sender, receivers, amount) {
+
+    let tx;
+    let txArray = [];
+    let batch;
+    let batchTx;
+    let senderDid = await api.query.identity.keyToIdentityIds(sender.publicKey);
+    let batchSize = 10;
```

Comment:
Consider not upfront declaring stuff here...

Reply:
understood, made the changes.